consensus/types/src/chain_spec.rs

```diff
@@ -972,6 +1019,9 @@ pub struct Config {
     ejection_balance: u64,
     #[serde(with = "serde_utils::quoted_u64")]
     min_per_epoch_churn_limit: u64,
+    #[serde(default)]
```

Comment:
This should probably default to something other than 0 if omitted.

Reply:
updated to 8 here: 7060bb6a76ad04aa91f3a262197865a69c0fdb3a